node-notes-cli/app.js

```diff
@@ -0,0 +1,25 @@
+const data = require('./data.json')
+const fs = require('fs')
+
+function command() {
+  if (process.argv[2] === 'read') {
+    for (const item in data.notes) {
+      console.log(`${item}: ${data.notes[item]}`)
+    }
+  }
+  if (process.argv[2] === 'create') {
+    data.notes[data.nextId] = process.argv[3];
+    data.nextId++;
+  }
+  if (process.argv[2] === 'delete') {
+    delete data.notes[process.argv[3]];
+  }
+  if (process.argv[2] === 'update') {
+    data.notes[process.argv[3]] = process.argv[4];
+  }
```

Comment:
"User can update a note" shouldn't be able to add a new note by accident. How would you prevent new notes from being added during an update?

Reply:
Updated the code with a condition to check if the note ID exists before use can make updates. Please review. Thanks.
![image](https://user-images.githubusercontent.com/76827487/128430742-0a8aaec1-05a6-4244-b5c6-f018f231d4f9.png)